packages/react/src/components/dialog/style.css.ts

```diff
@@ -0,0 +1,58 @@
+import { keyframes, style } from "@vanilla-extract/css";
+import { publicVariables, theme } from "../../theme/theme.css";
+
+const show = keyframes({
+  "0%": { opacity: "0" },
+  "100%": { opacity: "1" },
+});
+
+const animation = `${show} 150ms cubic-bezier(0.16, 1, 0.3, 1)`;
+
+export const overlay = style({
+  backgroundColor: publicVariables.color.placeholder,
+  position: "fixed",
+  inset: "0",
+  animation,
+});
+
+export const wrapper = style({
+  position: "fixed",
+  top: "50%",
+  left: "50%",
+  transform: "translate(-50%, -50%)",
+  width: "90vw",
+  maxWidth: "398px",
```

Comment:
Can we transition over the height property so the dialog does not jump when the content changes height?